view/sharedcache/core/SlideInfo.cpp

```diff
@@ -0,0 +1,304 @@
+#include "SlideInfo.h"
+
+#include "Dyld.h"
+#include "SharedCache.h"
+
+#ifdef _MSC_VER
+int count_trailing_zeros(uint64_t value) {
```

Comment:
These helper functions and the other non member functions in this file should be either static or in an unnamed namespace so they're internal to this translation unit.